packages/stack/src/lib/stack-app.ts

```diff
@@ -2489,6 +2489,13 @@ class _StackAdminAppImpl<HasTokenStore extends boolean, ProjectId extends string
   protected async _refreshApiKeys() {
     await this._apiKeysCache.refresh([]);
   }
+
+  async sendAdminRequest(
+    path: string,
+    requestOptions: RequestInit,
+  ) {
+    return await this._interface.sendAdminRequest(path, requestOptions, await this._getSession());
+  }
```

Comment:
this should be on `stackApp[stackAppInternalsSymbol].sendAdminRequest` (like the other `sendRequest`), as it shouldn't be exposed to anyone but the dashboard (optimally we can just override `sendRequest` instead, and we should do the same for the server app)